src/context/frame_header.rs

```diff
@@ -171,42 +210,6 @@ impl<'a> ContextWriter<'a> {
       }
     }
   }
-  // rather than test writing and rolling back the cdf, we just count Q8 bits using the current cdf
-  pub fn count_lrf_switchable(
-    &self, w: &dyn Writer, rs: &TileRestorationState,
-    filter: RestorationFilter, pli: usize,
-  ) -> u32 {
-    let nsym = &self.fc.lrf_switchable_cdf.len() - 1;
```

Comment:
what about keeping it and call `self.fc.count_lrf_switchable` from there?

Reply:
Do you mean so that we don't have to write cw.fc.count_lrf_switchable when not constrained by the borrow checker?

Reply:
yes :)